cs3/gateway/v1beta1/gateway_api.proto

```diff
@@ -308,6 +309,8 @@ service GatewayAPI {
 
   // Creates a new ocm share.
   rpc CreateOCMCoreShare(cs3.ocm.core.v1beta1.CreateOCMCoreShareRequest) returns (cs3.ocm.core.v1beta1.CreateOCMCoreShareResponse);
+  // Returns the iframe url.
+  rpc Open(cs3.app.provider.v1beta1.OpenRequest) returns (cs3.app.provider.v1beta1.OpenResponse);
```

Comment:
I would go for a more descriptive name of what the method actually `Open`s  (e.g. `OpenFileInAppProvider()`) and a bit of context on where the iframe URL comes from.

Reply:
Also, is it a part of the "`OCM CORE`" as the block comment mentions on top? Does it make sense to create a new block for "`APP PROVIDERS`"?

Reply:
Good points, will update this!

Reply:
Thank you for the comments. I agree that `OpenFileInAppProvider` is a good name! WHat do you think @glpatcern?

Reply:
Yes! for an API, I like it being verbose.
Also the return URL - being it iframed or not - should reflect the fact that it comes from an external Application. `AppProviderURL`? Any better suggestion?

Reply:
👍 I'll update this